05/ContactForm.js

```diff
@@ -1,9 +1,11 @@
-import React, { useReducer } from 'react';
+import React, { useReducer, useState } from 'react';
 
 // import account from './account';
-import { UPDATE_FORM, onInputChange, onFocusOut } from './lib/formUtils';
+import { UPDATE_FORM, onInputChange, onFocusOut, validateInput } from './lib/formUtils';
 
 const ContactForm = () => {
+    const [showError, setShowError] = useState(false);
+
     const initialState = {
         name: { value: '', touched: false, hasError: true, error: '' },
```

Comment:
Może zamiast pisać 2 właściwości tj. `showError` oraz `error` to może mógłbyś tylko sprawdzać czy `error === ''` (nie pokazuj błędu) lub `error !== ''` (pokaż błąd)